07-documentation.md

```diff
@@ -0,0 +1,80 @@
+# REST documentation
+
```

Comment:
Will this entire chapter be impacted by your investigation of next-gen rest documentation tools?